docs/import/add_disaster.js

```diff
@@ -327,6 +352,7 @@ function initializeScoreSelectors(states) {
  * @param {Array<string>} assets array of assets for add to dropdown
  * @param {string} row The asset type/row to put the dropdown in.
  * @param {string} state The state the assets are in.
+ * @return {JQuery<HTMLSelectElement>}
  */
 function createAssetDropdown(assets, row, state) {
```

Comment:
I think this function is broken now, because it's iterating up to assets.length which is now undefined. May not matter that much, since I have to modify this function a lot, but might be nice to keep it un-broken. Also, for score assets, everything will be a feature collection. I wonder if we could thread the needle in terms of display by leaving out the FEATURE_COLLECTION modifier in general, assuming it's the default? Maybe not worth it.

Reply:
Yeah, I think it won't be super-useful for them, but not a big deal